modules/nf-core/gatk4/filterintervals/main.nf

```diff
@@ -0,0 +1,47 @@
+process GATK4_FILTERINTERVALS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::gatk4=4.3.0.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.3.0.0--py36hdfd78af_0':
+        'quay.io/biocontainers/gatk4:4.3.0.0--py36hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(intervals)
```

Comment:
since this can take in intervals/tsv/hdf5, perhaps we should call this input instead of intervals?

Reply:
Fair point!